walkthroughs/eks-canary-deployments-pipeline/shared_stack/lambda_functions/deploy_and_switch_traffic/bootstrap

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -euo pipefail
+
+export KUBECONFIG=/tmp/kubeconfig
+export PATH=$PATH:/opt/awscli:/opt/kubectl
+
+# Handler format: <script_name>.<bash_function_name>
+#
+# The script file <script_name>.sh  must be located at the root of your
+# function's deployment package, alongside this bootstrap executable.
+source $(dirname "$0")/"$(echo $_HANDLER | cut -d. -f1).sh"
+
+while true
+do
+    # Request the next event from the Lambda runtime
+    HEADERS="$(mktemp)"
+    EVENT_DATA=$(curl -sS -LD "$HEADERS" -X GET "http://${AWS_LAMBDA_RUNTIME_API}/2018-06-01/runtime/invocation/next")
+    INVOCATION_ID=$(grep -Fi Lambda-Runtime-Aws-Request-Id "$HEADERS" | tr -d '[:space:]' | cut -d: -f2)
+
+    handler $EVENT_DATA
+    # Execute the handler function from the script
+    #RESPONSE=$($(echo "$_HANDLER" | cut -d. -f2) "$EVENT_DATA")
+    #RESPONSE="OK"
+    # Send the response to Lambda runtime
+    curl -sS -X POST "http://${AWS_LAMBDA_RUNTIME_API}/2018-06-01/runtime/invocation/$INVOCATION_ID/response" -d "$response"
```

Comment:
`$response` is sent via the `POST` request but is undefined given the comment.

Reply:
Removed commented lines in commit [384424a](https://github.com/aws/aws-app-mesh-examples/pull/348/commits/384424a17bd8c9243c31c78a974d9e0a91c66f9c) to prevent confusion